test/integration/patients/worklist/schedule.js

```diff
@@ -205,6 +205,44 @@ context('schedule page', function() {
       .should('contain', '2:00 PM')
       .should('contain', 'Third Action');
 
+    cy
+      .get('@actionList')
+      .find('tr')
+      .last()
+      .find('.js-patient-sidebar-button')
+      .click();
+
+    cy
+      .get('.app-frame__sidebar .sidebar')
+      .find('.worklist-patient-sidebar__patient-name')
+      .should('contain', 'Test Patient');
+
+    cy
+      .get('[data-filters-region]')
+      .find('[data-owner-filter-region]')
+      .click();
+
+    cy
+      .get('.picklist')
+      .find('.picklist__group')
+      .eq(1)
+      .click();
+
+    cy
+      .get('.app-frame__sidebar .sidebar')
+      .should('not.exist');
+
+    cy
+      .get('[data-filters-region]')
+      .find('[data-owner-filter-region]')
+      .click();
+
+    cy
+      .get('.picklist')
+      .find('.picklist__group')
+      .contains('Clinician McTester')
+      .click();
+
```

Comment:
This test does the following:

* Opens the patient sidebar app from the schedule page and verifies that it contains the correct patient name.
* Verifies that the patient sidebar closes when a filter is applied to the list.

We test the contents of the patient sidebar extensively in the worklist tests, so I figured we don't need to do that again here for the schedule.